knowledge-base/customize-selection-indicator-color-navigationview.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Customizing Selection Indicator Color in NavigationView for .NET MAUI on Windows
+description: Learn how to change the selection indicator color of NavigationViewItemView in .NET MAUI NavigationView on Windows.
+type: how-to
+page_title: How to Customize the Selection Indicator Color in .NET MAUI NavigationView on Windows.
+slug: customize-selection-indicator-color-navigationview-.net-maui
+tags: navigationview, .net maui, selection indicator, color, customization
+res_type: kb
+ticketid: 1656066
+---
+
+## Environment
+
+| Product | Version |
+| --- | --- |
+| Telerik UI for .NET MAUI | Current |
```

Comment:
```suggestion
| Telerik UI for .NET MAUI NavigationView | Latest |
```

Reply:
in previous kb articles we list the exact number of the version in which the kb is generated. So I prefer using the telerik maui version instead `latest` and `current` keywords.